action/__tests__/plugins/staticHosting/helpers.test.ts

```diff
@@ -0,0 +1,47 @@
+import * as core from '@actions/core';
+import { afterEach, beforeEach, describe, expect, it, jest } from '@jest/globals';
+import fetch, { Response } from 'cross-fetch';
+
+import { uploadFileWithSignedUrl } from '../../../src/plugins/staticHosting/helpers';
+
+jest.mock('cross-fetch', () => jest.fn());
+const mockFetch = fetch as jest.MockedFunction<typeof fetch>;
+
+jest.mock('@actions/core');
+const mockCore = core as jest.Mocked<typeof core>;
+
+describe('uploadFileWithSignedUrl', function () {
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
+
+  describe('when there is no error', function () {
+    it('uploads the file with one fetch call', async () => {
+      mockFetch.mockResolvedValue({ ok: true } as Response);
```

Comment:
Should move this setup to a "beforeEach" block to be consistent across different tests within this file.